tests/test_e2e.py

```diff
@@ -0,0 +1,68 @@
+from threading import Thread
+from selenium import webdriver
+from dotenv import load_dotenv, find_dotenv
+from selenium.webdriver.common.keys import Keys
+import os
+import pymongo
+import pytest
+import app
+import random
+import string
+
+def get_db_collection():
+
+    dbClientUri = os.getenv('MONGO_DB_CONNECTION')
+    databaseName = os.getenv('MONGO_DB_DATABASE')
+    collectionName = 'collection'
+
+    dbClient = pymongo.MongoClient(dbClientUri)
+    db = dbClient[databaseName]
+    return db[collectionName]
+    
+@pytest.fixture(scope='module')
+def test_app():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
```

Comment:
As mentioned above your e2e tests need real credentials and `env.test` should have fake credentials in. You can just delete these two lines as long as your CI and readme instructions load environment variables another way (which it looks like they already do)